Code/BuildSystem/CMake/ezUtilsNuGet.cmake

```diff
@@ -0,0 +1,24 @@
+######################################
+### NuGet support
+######################################
+
+######################################
+### ez_requires_nuget()
+######################################
+
+function(ez_requires_nuget)
```

Comment:
My "requires" functions meant to exclude the project, if X is not available.
Could you rename this?

Reply:
Good point, renamed it to ez_nuget_init(). The function should never fail unless MS server goes down or you don't have internet.